examples/sprite_batch/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+
+set(SRCROOT ${PROJECT_SOURCE_DIR}/examples/sprite_batch)
+
+# all source files
+set(SRC ${SRCROOT}/Batching.cpp)
+if (SFML_OS_IOS)
+    set(RESOURCES
+        ${CMAKE_CURRENT_SOURCE_DIR}/resources/Tex0.png
+        ${CMAKE_CURRENT_SOURCE_DIR}/resources/Tex1.png
+        ${CMAKE_CURRENT_SOURCE_DIR}/resources/Tex2.png
+        ${CMAKE_CURRENT_SOURCE_DIR}/resources/Tex3.png
+        ${CMAKE_CURRENT_SOURCE_DIR}/resources/Tex4.png
+        ${CMAKE_CURRENT_SOURCE_DIR}/resources/Tex5.png
+        ${CMAKE_CURRENT_SOURCE_DIR}/resources/Tex6.png
+        ${CMAKE_CURRENT_SOURCE_DIR}/resources/Tex7.png
+        ${CMAKE_CURRENT_SOURCE_DIR}/resources/Tex8.png)
+    set_source_files_properties(${RESOURCES} PROPERTIES MACOSX_PACKAGE_LOCATION Resources)
+endif()
+
+# define the pong target
```

Comment:
Pong target? I think you forgot to update this comment.

Reply:
Yeah, I just plagiarized the cmake script from that example project 😓